src/intrprtr.c

```diff
@@ -3989,16 +3881,6 @@ static Int InitKernel (
 }
 
 
-static Int InitModuleState(void)
-{
-    STATE(IntrCoding) = 0;
-    STATE(IntrIgnoring) = 0;
-    STATE(IntrReturning) = 0;
```

Comment:
The caller is now responsible for passing a zeroed `struct IntrState` to `IntrBegin`.